.gitlab-ci.yml

```diff
@@ -804,6 +804,19 @@ library:ci-deriving:
   - build:edge+flambda
   - library:ci-mathcomp
 
+library:ci-mathcomp_word:
+  extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - library:ci-mathcomp
```

Comment:
```suggestion
  - build:edge+flambda
  - library:ci-mathcomp_base
  - library:ci-mathcomp
```

Reply:
No, mathcomp_base is mathcomp for those using non-flambda